src/legacy/core_plugins/data/public/search/aggs/buckets/_terms_other_bucket_helper.ts

```diff
@@ -20,18 +20,21 @@
 import _ from 'lodash';
```

Comment:
nit: Since we are already refactoring a bit here, might be nice to explicitly import the functions we need from lodash, as that's the preferred convention, e.g.

```js
import { values, ... } from 'lodash';
```